types/frisby/index.d.ts

```diff
@@ -1,75 +1,234 @@
-// Type definitions for Frisby v0.8.5
+// Type definitions for Frisby 2.0
 // Project: https://github.com/vlucas/frisby
-// Definitions by: Johnny Li <https://github.com/johnny4753>
+// Definitions by: Christopher E. Woodland <https://github.com/cwoodland>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.6
 
-///<reference types="jasmine"/>
+/// <reference types='jasmine'/>
 
-declare interface Frisby {
-    globalSetup(opts: any): any;
-    timeout(seconds?: number): Frisby;
-    reset(): Frisby;
-    not(): Frisby;
-    addHeader(key: string, content: string): Frisby;
-    addHeaders(headers: object): Frisby;
-    removeHeader(key: string): Frisby;
-    responseType(type: any): Frisby;
-    // HTTP Basic Auth
-    auth(username: string, password: string, digest: boolean): Frisby;
-    // HTTP Request
-    get(uri: string, params?: any): Frisby;
-    patch(uri: string, data?: any, params?: any): Frisby;
-    post(uri: string, data?: any, params?: any): Frisby;
-    put(uri: string, data?: any, params?: any): Frisby;
-    delete(uri: string, data?: any, params?: any): Frisby;
-    head(uri: string, params?: any): Frisby;
-    options(uri: string, params?: any): Frisby;
+// #region Imports
+import { Response } from 'node-fetch'; // Import all definitions from node-fetch.
```

Comment:
As you have it now, this line isn't doing anything (i.e., it's not used within the declaration file and it's not accessible outside of it).

Reply:
Frisby.js exports a class called FrisbySpec.
FrisbySpech has a method promise which returns a node-fetch Response.
https://github.com/vlucas/frisby/blob/master/src/frisby/spec.js#L258

I have changed the line to  export import nodeFetch = require('node-fetch'); 
And changed line 41 to promise(): Promise<nodeFetch.Response>;